winit-core/winit_tests/hello_world/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+// MIT/Apache2 License
+
+use futures_lite::future::block_on;
+
+#[allow(clippy::eq_op)]
+fn main() {
+    winit_test::run_tests(|harness| {
+        block_on(async move {
+            harness.test("hello world", async {
+                assert_eq!(1 + 1, 2)
+            }).await;
+        });
```

Comment:
Do we really need any tests here? This doesn't have any core logic stuff it should all go into the `winit` crate instead.